src/main/java/org/folio/helper/RestoreHelper.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.helper;
+
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import org.folio.rest.core.models.RequestContext;
+import org.folio.rest.jaxrs.model.Piece;
+import org.folio.rest.jaxrs.model.ReceivingCollection;
+import org.folio.rest.jaxrs.model.ReceivingResults;
+
+import java.util.Map;
+
+public class RestoreHelper extends ReceivingHelper {
```

Comment:
**Doubt**
By extending some class we inherit not only methods but state as well. Having global fields in service/helper classes is a separate issue but now we are adding a lot of compelcity on top. Let's use aggregation if possible:
`RestoreHelper` can inject `ReceivingHelper`.